Description: fix manpages
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 wifi-qr (0.1-1) unstable; urgency=medium
 .
   * Initial release
Author: kokoye2007 <kokoye2007@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-05-31

--- /dev/null
+++ wifi-qr-0.1/wifi-qr.1
@@ -0,0 +1,64 @@
+.\"                                      Hey, EMACS: -*- nroff -*-
+.\" (C) Copyright 2019 kokoye2007 <kokoye2007@gmail.com>,
+.\"
+.\" First parameter, NAME, should be all caps
+.\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
+.\" other parameters are allowed: see man(7), man(1)
+.TH WIFI-QR 1 "MAY 31 2020"
+.\" Please adjust this date whenever revising the manpage.
+.\"
+.\" Some roff macros, for reference:
+.\" .nh        disable hyphenation
+.\" .hy        enable hyphenation
+.\" .ad l      left justify
+.\" .ad b      justify to both left and right margins
+.\" .nf        disable filling
+.\" .fi        enable filling
+.\" .br        insert line break
+.\" .sp <n>    insert n+1 empty lines
+.\" for manpage-specific macros, see man(7)
+.SH NAME
+wifi-qr \-  WiFi Share and Connect with QR
+.SH SYNOPSIS
+.B wifi-qr
+.RI [ Gg | Cc | Tt | Ss | Qq | Vv ]  
+.br
+.SH DESCRIPTION
+ PC to Android Network SSID Password share with QR code.
+ Idea from Xiaomi Android WiFi Password share with QR
+ Its just bash script and qrencode only.
+ now support also QR Scan with connect to WIFI.
+ Make to easy and happy.
+.sp
+
+.PP
+.SH OPTIONS
+wifi-qr option  
+.TP
+.B g 	GUI Main Menu 
+.br
+.TP
+.B c 	WiFi QR Create GUI
+.br
+.TP
+.B t 	WiFi QR Create Terminal
+.br
+.TP
+.B s 	QR Scan and Auto Connect WiFi
+.br
+.TP
+.B q 	QR Scan and Connect WiFi GUI
+.br
+.TP
+.B v 	WiFi-QR Version is 0.1.1
+.br
+
+
+.SH BUGS
+If there are any, please notify the author at the address below.
+.SH AUTHOR
+kokoye2007 <kokoye2007@gmail.com>
+
+.SH SEE ALSO
+.BR qrencode (1), nmcli (1), zbarcam (1), zbarimg (1)
+.br
--- wifi-qr-0.1.orig/wifi-qr.desktop
+++ wifi-qr-0.1/wifi-qr.desktop
@@ -1,12 +1,11 @@
 [Desktop Entry]
 Type=Application
 Version=0.1
-Encoding=UTF-8
-Name=Wifi QR
+Name=WiFi QR
 Comment=Wifi share with QR Code
 Exec=sh -c 'pkexec wifi-qr g'
 Icon=wifi-qr
-Keywords="QR; Wifi; Password; Share;"
+Keywords="QR; WiFi; Password; Share;"
 Terminal=false
 Categories=GNOME;Utility;
 Actions=ScanQR;CreateQR;
